### DISCUSSION TITLE
Windows 11 now Supports DoH

### DISCUSSION THREAD

**Discussion by irrweg** [AdguardTeam/AdGuardHome#5380] [General]
Greetings,

I recently saw in the Network Settings that you can Setup DNS Servers using DoH as Encryption.
Would be great to update the Setup Guide Page of Windows to make people aware about that option.

No need for third-party apps to set Encrypted DNS Request, except you want DoT or DNSCrypt.

My Windows 11 Version is 22H2.

![Screenshot 2023-01-19 205804](https://user-images.githubusercontent.com/345180/213548076-4e6c0a06-c9c3-4770-be0a-3e067053ee16.jpg)
![Screenshot 2023-01-19 205615-1](https://user-images.githubusercontent.com/345180/213548070-4a7c21ec-40a8-4085-a048-a3b14a2e9e0e.jpg)

**Comment by ammnt** [comment]
@ploedman, this feature request is already exist:
https://github.com/AdguardTeam/AdGuardHome/issues/3709

Will be implemented soon I hope🤲🏼
